src/components/PathwaysList/PathwayModal.tsx

```diff
@@ -49,6 +52,30 @@ const NewPathwayModal: FC<NewPathwayModalProps> = ({ open, onClose }) => {
     [addPathway, closeModal]
   );
 
+  const handleUpdatePathway = useCallback(
+    (event: FormEvent<HTMLFormElement>): void => {
+      event.preventDefault();
+      if (editPath) {
```

Comment:
Is this here just because of typescript warnings? It shouldn't be needed since this function is called from `createNewModal ? handleCreateNewPathway : handleUpdatePathway` and `createNewModal = !editPath`

Reply:
Agree that it shouldn't be needed, but it doesn't compile without it. I couldn't figure out why the object was being flagged as possibly undefined. It might be because editPath is in the dependency list, so maybe it's type checking because of potential changes elsewhere to editPath?